ietf-layer0-types.yang

```diff
@@ -707,30 +796,51 @@ module ietf-layer0-types {
       type empty;
     }
     description
-      "A decimal64 value with two digits, when the value is know or
+      "A decimal64 value with two digits, when the value is known or
       an empty value when the value is not known.";
   }
 
-  typedef power-in-db {
-    type decimal-2-digits;
-    units dB;
+  typedef gain-in-db {
+    type decimal-2-digits {
+      range "0..max";
+    }
+    units "dB";
     description
-      "The power in dB.";
+      "The power gain in dB.";
```

Comment:
I would suggest to remove the word power: "The gain in dB.";

Reply:
Addressed in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/70/commits/0038f5e771db23c9e2109f149b30d3226340db67

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>